client/env.d.ts

```diff
@@ -3,6 +3,9 @@
 interface ImportMetaEnv {
   readonly VITE_BASE_TZ: string;
 }
+interface ImportMetaEnv {
+  readonly VITE_BASE_URL: string;
```

Comment:
Wo brauchen wir das überhaupt?
Ich habe das weder in meinem lokalen env definiert noch auf dem Live-Server ist das vorhanden

Reply:
Hättest ja mal danach suchen können im Code:P
Wurde mal bei der routerHistory verwendet, kann aber weg. (Hab ich schon gemacht.)

Reply:
Habe ich ;-)

Aber die Idee dahinter war mir trotzdem nicht klar